### PR TITLE
pin linter execution in github actions to specific go version

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*
Checking if we can set goversion this way for the linter

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -11,8 +11,12 @@ jobs:
...
     steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.16.15
       - uses: actions/checkout@v2
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
           version: latest
+          skip-go-installation: true
```
Comment (chrisdoherty4) [inline .github/workflows/golangci-lint.yml:22]: I think this is redundant. See https://github.com/golangci/golangci-lint-action#compatibility

**Review by chrisdoherty4 — APPROVED** [review]

File: .github/workflows/golangci-lint.yml (hunk 1 above)
Comment (danbudris) [reply .github/workflows/golangci-lint.yml:22]: ah I see

Comment (danbudris) [reply .github/workflows/golangci-lint.yml:22]: turns out it looks like we need this

**Review by abhinavmpandey08 — APPROVED** [review]
/lgtm

**Pull request merged**